static/app/components/charts/optionSelector.tsx

```diff
@@ -47,6 +47,7 @@ function OptionSelector({
   const mappedOptions = useMemo(() => {
     return options.map(opt => ({
       ...opt,
+      textValue: opt.label,
```

Comment:
Looks like you have to cast to string here so TS will be happy?

Reply:
Oh true somehow I missed that! Just fixed it